static/src/stylesheets/layout/new-header/_pillar-link.scss

```diff
@@ -5,7 +5,7 @@
     cursor: pointer;
     display: block;
     font-size: 16px;
-    font-size: 5.1vw;
+    font-size: 4.9vw;
```

Comment:
Thing is there is very very little space in this area and I can't think about any way how to make this way with bigger fonts. Can add it to the list of things to look into ...